File: scripts/updateContacts.mjs
import { readFileSync, writeFileSync } from 'fs';

// Read the JSON file
const data = JSON.parse(readFileSync('app/data/kenyanMarketProducts.json', 'utf8'));

// Update all supplier contacts
let updated = 0;
for (const category in data) {
  if (Array.isArray(data[category])) {
    data[category].forEach(item => {
      if (item.supplier) {
        item.supplier = 'EmersonEIMS';
        item.supplierContact = '+254 768 860 655 | +254 782 914 717';
        updated++;
      }
    });
  }
}

// Write back to file
writeFileSync('app/data/kenyanMarketProducts.json', JSON.stringify(data, null, 2), 'utf8');

console.log(`✅ Updated ${updated} products with EmersonEIMS contact`);
console.log('   Phone: +254 768 860 655 | +254 782 914 717');
console.log('   Email: emersoneimservices@gmail.com | info@emersoneims.com');

File: scripts/updateContacts.py
import json

# Read the JSON file
with open('app/data/kenyanMarketProducts.json', 'r', encoding='utf-8-sig') as f:
    data = json.load(f)

# Update all supplier contacts
for category_name, category_data in data.items():
    if isinstance(category_data, list):
        for item in category_data:
            if 'supplier' in item:
                item['supplier'] = 'EmersonEIMS'
                item['supplierContact'] = '+254 768 860 655 | +254 782 914 717'

# Write back to file
with open('app/data/kenyanMarketProducts.json', 'w', encoding='utf-8') as f:
    json.dump(data, f, indent=2, ensure_ascii=False)

print("✅ All supplier contacts updated to EmersonEIMS")
print("   Phone: +254 768 860 655 | +254 782 914 717")
print("   Email: emersoneimservices@gmail.com | info@emersoneims.com")
